src/css/styles.css

```diff
@@ -91,7 +97,52 @@ figure {
 
 .user-steps,
 .user-sleep,
-.user-hydration {
+.user-hydration,
+.user-friends {
   display: flex;
   flex-direction: column;
+}
+
+.user-friend {
+  list-style-type: none;
+  padding: 0;
+}
+
+.user-name-greeting {
+  color: #ff761fde;
+  font-size: 48px;
+}
+
+.orange {
+  color: #ff761fde;
+}
+
+.greeting {
+  display: flex;
+  flex-direction: column;
+  justify-content: flex-start;
+  align-items: center;
+}
+
+.greetings {
+  font-size: 18px;
+  margin: 2px;
```

Comment:
I believe this margin is actually not being displayed in the browser, so it's not needed.